src/transitions/transition-ios.ts

```diff
@@ -9,6 +9,8 @@ const TRANSFORM = 'transform';
 const TRANSLATEX = 'translateX';
 const OFF_RIGHT = '99.5%';
 const OFF_LEFT = '-33%';
+const OFF_RIGHT_RTL = '-99.5%';
+const OFF_LEFT_RTL = '33%';
```

Comment:
what do think if you use `OFF_RIGHT` --> `OFF_START`, `OFF_LEFT` --> `OFF_END`  ? 
then you use the check `_plt.isRTL` just one time. Also you would  help the framwork to adapt `Start` and `End` notations 
````js
OFF_START = _plt.isRTL ? '-99.5%' : '99.5%';
OFF_END = _plt.isRTL ? '33%' : '-33%';
`````